NetworkingTests/Tests/EntityRepositorySpec.swift

```diff
@@ -154,6 +191,23 @@ class EntityRepositorySpec: QuickSpec {
             
         }
         
+        describe("#fetchCustomFailingEntityMishandlingError") {
+            
+            it("fetches a single entity from JSON file and fails with a custom error mishandled") { waitUntil { done in
+                repository.fetchCustomFailingEntityMishandlingError().startWithResult {
+                    switch $0 {
+                    case .success: fail()
+                    case .failure(let error):
+                        switch error {
+                        case .requestError: done()
```

Comment:
Probably it'd be better if I move them to this spec. I had done that before adding this new class, that's why they're there.